lib/commands/deploy/helper.js

```diff
@@ -1,16 +1,27 @@
 const SkillMetadataController = require('@src/controllers/skill-metadata-controller');
 const SkillCodeController = require('@src/controllers/skill-code-controller');
 const SkillInfrastructureController = require('@src/controllers/skill-infrastructure-controller');
+const ResourcesConfig = require('@src/model/resources-config');
 const profileHelper = require('@src/utils/profile-helper');
+const CONSTANTS = require('@src/utils/constants');
 const Messenger = require('@src/view/messenger');
 
 module.exports = {
+    highlightProfile,
     deploySkillMetadata,
     buildSkillCode,
     deploySkillInfrastructure,
     enableSkill
 };
 
+function highlightProfile(profile) {
+    const profileList = ResourcesConfig.getInstance().getProfileList();
+    if (!profileList.includes(profile)) {
+        throw `Profile [${profile}] does not exist. Please configure it in your ${CONSTANTS.FILE_PATH.ASK_RESOURCES_JSON_CONFIG} file.`;
+    }
+    Messenger.getInstance().info(`Deploy project for profile [${profile}]\n`);
```

Comment:
Probably should be outside of this method since this methods responsibility is just to check that profile exists.